person.rb

```diff
@@ -1,14 +1,16 @@
 require './corrector'
 
 class Person
-  attr_writer :id
-  attr_accessor :name, :age
+  attr_reader :id
+  attr_accessor :name, :age, :rentals
 
-  def initialize(age, _name: 'Unknown', _parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] **[OPTIONAL]** I am still unable to create a person because you did not use the `=` sign for your **name** parameter, as highlighted previously, like so: `name = 'Unknown'`. It would be nice to fix this now, before moving forward in the projects.

Reply:
@TedLivist,

Thank you for the feedback. But I am able to create a person on my system and I am just wondering why you are unable to do so on your system. Kindly state the error received maybe that will help.

Reply:
@micheaol, here is the initialize method as it is:
  ```
  def initialize(age, name: 'Unknown', parent_permission: true)
  ```
I get the error as shown in the image below.

![ola-corr](https://user-images.githubusercontent.com/36542986/151277451-57129eb2-773d-4c89-b666-1e779c0f30e4.jpg)

Reply:
@TedLivist ,
Thank you for your reply. I will look into this.